src/test/java/org/minions/devfund/sergio/SquarelotronTest.java

```diff
@@ -0,0 +1,397 @@
+package org.minions.devfund.sergio;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test methods for Squarelotron.
+ *
+ * @author Sergio Navarro
+ * @since 4/20/2018
+ */
+public class SquarelotronTest {
+
+    private StringBuilder test = new StringBuilder();
+    private Squarelotron squarelotron;
+
+    /**
+     * Tests the values in the square of 2x2.
+     */
+    @Test
+    public void testSetValuesForSize2x2() {
+        final int size = 2;
+        squarelotron = new Squarelotron(size);
+        test.append("01 02").append("\n")
+                .append("03 04");
+        assertEquals(squarelotron.showSquare(squarelotron.getOriginalSquareArray()), test.toString().trim());
+    }
+
+    /**
+     * Tests the values in the square of 5x5.
+     */
+    @Test
+    public void testSetValuesForSize5x5() {
+        final int size = 5;
+        squarelotron = new Squarelotron(size);
+        test.append("01 02 03 04 05").append("\n")
+                .append("06 07 08 09 10").append("\n")
+                .append("11 12 13 14 15").append("\n")
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "11 12 13 14 15" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)